ui/apps/vue-mri-ui-lib/src/styles/app.scss

```diff
@@ -5,6 +5,10 @@ html,
 body,
 #app {
   height: 100%;
+  max-width: 100vw !important;
```

Comment:
Not necessary needed. This can be removed after the @single-spa/welcome is removed from the `VUE_APP_NAVIGATION_ITEMS` as it override the body styles.